docs/mobile_data_collected.md

```diff
@@ -0,0 +1,150 @@
+|-----------------|--------|-------------------------------------------------------------------|
+| `error.source`  | string | Where the error originates from (for example, `console` or `network`).     |
+| `error.type`    | string | The error type (or error code in some cases).                   |
+| `error.message` | string | A concise, human-readable, one-line message explaining the event. |
+| `error.stack`   | string | The stack trace or complementary information about the error.     |
+
+#### Network errors
+
+Network errors include information about failing HTTP requests. The following facets are also collected:
+
+| Attribute                      | Type   | Description                                                                             |
+|--------------------------------|--------|-----------------------------------------------------------------------------------------|
+| `error.resource.status_code`             | number | The response status code.                                                               |
+| `error.resource.method`                | string | The HTTP method (for example `POST`, `GET`).           |
+| `error.resource.url`                     | string | The resource URL.                                                                       |
+| `error.resource.provider.name`      | string | The resource provider name. Default is `unknown`.                                            |
+| `error.resource.provider.domain`      | string | The resource provider domain.                                            |
+| `error.resource.provider.type`      | string | The resource provider type (for example `first-party`, `cdn`, `ad`, `analytics`).                                            |
+
+
+{{% /tab %}}
+{{% tab "User Action" %}}
+
+
+
+### Action timing metrics
+
+| Metric    | Type   | Description              |
+|--------------|--------|--------------------------|
+| `action.loading_time` | number (ns) | The loading time of the action. See how it is calculated in the [User Action documentation][4]. |
+| `action.long_task.count`        | number      | Count of all long tasks collected for this action. |
+| `action.resource.count`         | number      | Count of all resources collected for this action. |
+| `action.error.count`      | number      | Count of all errors collected for this action.|
+
+### Action attributes
+
+| Attribute    | Type   | Description              |
+|--------------|--------|--------------------------|
+| `action.id` | string | UUID of the user action. |
+| `action.type` | string | Type of the user action. For [Custom User Actions][5], it is set to `custom`. |
+| `action.target.name` | string | Element that the user interacted with. Only for automatically collected actions |
+
+
+{{% /tab %}}
+{{< /tabs >}}
+
+
+## Data retention
+By default, all data collected is kept at full granularity for 15 days. 
+
+## Further Reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
```

Comment:
It looks like links are missing from this page. they need to be added here, as well as in any tabs that reference them.